### ISSUE TITLE
Lint error into a fresh project

### ISSUE THREAD

**Issue by alexbruno** [SoftboxLab/gandalf-lint-vue#4]
```bash
[alexbruno@manjaro-linux moo-cli-vue]$ npm run lint

> moo-cli-vue@0.1.0 lint /home/alexbruno/Documentos/web/github/moo-cli/moo-cli-vue
> vue-cli-service lint

 ERROR  Error: Cannot read config file: /home/alexbruno/Documentos/web/github/moo-cli/moo-cli-vue/node_modules/@softboxlab/eslint-config-gandalf-lint-vue/index.js
Error: Cannot find module 'plugin:vue/recommended'
Referenced from: /home/alexbruno/Documentos/web/github/moo-cli/moo-cli-vue/package.json
Error: Cannot read config file: /home/alexbruno/Documentos/web/github/moo-cli/moo-cli-vue/node_modules/@softboxlab/eslint-config-gandalf-lint-vue/index.js
Error: Cannot find module 'plugin:vue/recommended'
Referenced from: /home/alexbruno/Documentos/web/github/moo-cli/moo-cli-vue/package.json
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at resolve (internal/modules/cjs/helpers.js:30:19)
    at Array.map (<anonymous>)
    at Object.<anonymous> (/home/alexbruno/Documentos/web/github/moo-cli/moo-cli-vue/node_modules/@softboxlab/eslint-config-gandalf-lint-vue/index.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
```

**Comment by pablohpsilva** [comment]
Maybe @vilaboim can be a better help on this one, but... I'll give it a try.
Hold on tight.

**Comment by pablohpsilva** [comment]
Yeah... I created a PR #5 so @vilaboim can take a look at it and test it deeply.

**Comment by vilaboim** [comment]
@pablohpsilva I updated your PR #5 , take a look.

**Comment by pablohpsilva** [comment]
should be fixed on PR #5
